src/Language/PureScript/Interactive/Message.hs

```diff
@@ -37,18 +41,18 @@ prologueMessage = unlines
   , "Type :? for help"
   ]
 
+noInputMessage :: String
+noInputMessage = unlines
+  [ "purs repl: No input files; try running `pulp psci` instead."
+  , "For help getting started, visit " ++ guideURL
+  , "Usage: For basic information, try the `--help' option."
+  ]
+
 supportModuleMessage :: String
 supportModuleMessage = unlines
-  [ "PSCi requires the psci-support package to be installed."
-  , "You can install it using Bower as follows:"
-  , ""
-  , "  bower i purescript-psci-support --save-dev"
-  , ""
-  , "Or using psc-package:"
-  , ""
-  , "  psc-package install psci-support"
-  , ""
-  , "For help getting started, visit https://github.com/purescript/documentation/blob/master/PSCi.md"
+  [ "purs repl: psci-support is not loaded; try running `pulp psci` instead."
```

Comment:
To clarify, I realise we are trying to remove hardcoded assumptions of particular package managers in the compiler, but for me that doesn't need to extend as far as changing this message; it really is more directed at e.g. `psc-publish` which currently assumes that you are using Bower as your package manager and simply does not work otherwise. The aim of this is to ease a transition away from Bower, at least that's how I see it. So I don't think we really gain anything by removing this information from this error message.